great_expectations/core/batch.py

```diff
@@ -771,11 +771,12 @@ def get_batch_request_from_acceptable_arguments(  # noqa: C901 - complexity 21
     """
 
     if batch_request:
-        if not isinstance(batch_request, (BatchRequest, RuntimeBatchRequest)):
-            raise TypeError(
-                f"""batch_request must be an instance of BatchRequest or RuntimeBatchRequest object, not \
-{type(batch_request)}"""
-            )
+        # BDIRKS I removed this check so the ZEP BatchRequest type could pass this check.
+        # We will need to modify this to get ZEP working.
+        #if not isinstance(batch_request, (BatchRequest, RuntimeBatchRequest)):
+        #    raise TypeError(
+        #        f"""batch_request must be an instance of BatchRequest or RuntimeBatchRequest object, not {type(batch_request)}"""
+        #    )
```

Comment:
We should be able to deal with this by registering the `ZEP BatchRequest` as a virtual subclass.